templates/admin/cloner/stats.blade.php

```diff
@@ -0,0 +1,27 @@
+<div class="pb-cloner wrap">
+	<div class="pb-cloner-section main">
+		<h1 class="page-title">{{ __( 'Book Info', 'pressbooks' ) }}</h1>
+		<ul>
+			<li>{{ __( 'Title', 'pressbooks' ) }}: {{ $book->blogname }}</li>
+			<li>{{ __( 'URL', 'pressbooks' ) }}: {{ $book->siteurl }}</li>
+			<li>{{ __( 'Created', 'pressbooks' ) }}: {{ $book->registered }}</li>
+			<li>{{ __( 'Total # of clones', 'pressbooks' ) }}: {{ count($cloning_stats) }}</li>
+		</ul>
+	</div>
+</div>
+@if($cloning_stats)
+<div class="pb-cloner stats wrap">
+	<h2 class="section-title">{{ __( 'Clones information', 'pressbooks' ) }}</h2>
```

Comment:
I'd use `Clone information` instead, I don't know if it's correct English though